docs/SerhiiNSN/scripts/burger.js

```diff
@@ -0,0 +1,8 @@
+const burger = document.querySelector('.header__burger');
+const menu = document.querySelector('.nav');
+const body = document.querySelector('body');
+burger.addEventListener('click', () => {
+    burger.classList.toggle('active');
```

Comment:
Proly you can find more suitable names for these classes, like `opened`, `no-scroll` e.g.